src/current/v24.1/molt-fetch.md

```diff
@@ -424,7 +634,15 @@ molt fetch \
 --continuation-token '5e7c7173-101c-4539-9b8d-28fad37d0240'
 ~~~
 
-To retry all tables that failed, exclude `--continuation-token` from the command.
+To retry all tables that failed, exclude `--continuation-token` from the command. When prompted, type `y` to clear all active continuation tokens. To avoid the prompt (e.g., when running `molt fetch` in a job), include the `--non-interactive` flag:
```

Comment:
```suggestion
To retry all tables with failures, exclude `--continuation-token` from the command. When prompted, type `y` to clear all active continuation tokens. To avoid the prompt (e.g., when running `molt fetch` in a job), include the `--non-interactive` flag:
```